src/mumble/ACLEditor.cpp

```diff
@@ -104,6 +107,9 @@ ACLEditor::ACLEditor(int channelid, const MumbleProto::ACL &mea, QWidget *p) : Q
 	qsbChannelPosition->setRange(INT_MIN, INT_MAX);
 	qsbChannelPosition->setValue(pChannel->iPosition);
 
+	qsbChannelMaxUsers->setRange(0, INT_MAX);
+	qsbChannelMaxUsers->setValue(pChannel->uiMaxUsers);
```

Comment:
We should use http://doc.qt.io/qt-4.8/qabstractspinbox.html#specialValueText-prop to display "Unlimited" (or something like that) instead of 0.